instrumentation/kotlinx-coroutines/javaagent/src/test/kotlin/io/opentelemetry/javaagent/instrumentation/kotlinxcoroutines/KotlinCoroutinesInstrumentationTest.kt

```diff
@@ -493,6 +496,34 @@ class KotlinCoroutinesInstrumentationTest {
     )
   }
 
+  @Test
+  fun `test WithSpan annotation`() {
+    runBlocking {
+      annotated1()
+    }
```

Comment:
Can you add assertions for the trace produced by this?

Reply:
🤦 forgot the most important thing